### ISSUE TITLE
Defect: Lab 4 Exercise Form

### ISSUE THREAD

**Issue by heathermoses** [all-rit/ALL#545]
# Description

The accessibility buttons overlay the actual form in the Lab 4 exercise. The accessibility buttons should be beneath the form.

![Image](https://github.com/all-rit/ALL/assets/54957497/2ce1db7d-d6e4-44f6-84f4-675b78ed140a)
